spec/models/claim_spec.rb

```diff
@@ -610,6 +611,10 @@
       it "generates a reference" do
         expect(claim.reference).to_not eq nil
       end
+
+      it "sets the eligibility award amount" do
+        expect(claim.eligibility.attributes["award_amount"]).to be_a(BigDecimal)
```

Comment:
Why not `.to eq BigDecimal("2000.00")`

Reply:
I'm not sure I care what the exact amount is here as that is dependent on the inputs, which we're not specific about and we're in danger of mixing testing cocnerns. What I'm testing is that _an amount_ is set.

Reply:
Could go with:

```
expect(claim.eligibility.attributes["award_amount"]).to be_a(BigDecimal).and be_positive
```

Reply:
I'd go with just `be_positive` if what you want to test is that it's being set.